src/services/productsService.js

```diff
@@ -5,13 +5,23 @@ const getAllProducts = async (limit, offset) => {
     const products = await productsDao.getAllProducts(limit, offset);
     return products;
   } catch (err) {
-    console.log(err);
-    throw new Error(
-      "Error has occurred in getting All Products /productService/getAllProducts"
-    );
+    throw new Error("Error has occurred in getting All Products /productService/getAllProducts")
   }
 };
 
-module.exports = {
-  getAllProducts,
+const getProduct = async (productsId) => {
+  try {
+    if (!productsId) {
+      return res.status(404).json({ message: 'PRODUCTS_NOT_FOUND' });
+    }
+    const product = await productsDao.getProduct(productsId);
```

Comment:
productId의 존재 유무를 판단하는 것이 아닌, productId가 가르키는 데이터가 실제로 있는지 판단해야 합니다.
```suggestion
    const product = await productsDao.getProduct(productsId);
    
     if (!product) {
      return res.status(404).json({ message: 'PRODUCTS_NOT_FOUND' });
    }
    
```